misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -521,5 +519,15 @@ def get_job_state() -> str:
     raise ValueError("Job not found")
 
 
+def get_suite_name() -> str:
+    suite_name = os.getenv("BUILDKITE_LABEL", "Unknown Test")
+
+    retry_count = int(os.getenv("BUILDKITE_RETRY_COUNT", "0"))
+    if retry_count > 0:
+        suite_name += f" ({retry_count})"
```

Comment:
Should we change it to `(retry {retry_count})` or increment the value by one? (optional)